lib/src/components/camera/service.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:grpc/grpc.dart';
+import 'package:viam_sdk/src/gen/google/api/httpbody.pb.dart';
+
+import '../../gen/common/v1/common.pb.dart';
+import '../../gen/component/camera/v1/camera.pbgrpc.dart';
+import '../../media/image.dart';
+import '../../resource/manager.dart';
+import '../../utils.dart';
+import 'camera.dart';
+
+class CameraService extends CameraServiceBase {
+  final ResourceManager _manager;
+
+  CameraService(this._manager);
+
+  Camera _cameraFromManager(String name) {
+    try {
+      return _manager.getResource(Camera.getResourceName(name));
+    } catch (e) {
+      throw (GrpcError.notFound(e.toString()));
+    }
+  }
+
+  @override
+  Future<DoCommandResponse> doCommand(ServiceCall call, DoCommandRequest request) async {
+    Camera camera = _cameraFromManager(request.name);
+    var result = await camera.doCommand(request.command.toMap());
+    return DoCommandResponse(result: result.toStruct());
+  }
+
+  @override
+  Future<GetImageResponse> getImage(ServiceCall call, GetImageRequest request) async {
+    Camera camera = _cameraFromManager(request.name);
+    var result = await camera.image(mimeType: MimeType.fromString(request.mimeType));
+    return GetImageResponse(mimeType: result.mimeType.toString(), image: result.raw);
```

Comment:
I am unsure if `result.mimeType.toString()` will convert the mimeType like we want, I will verify when writing the tests.